benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataLoader.java

```diff
@@ -83,19 +85,14 @@ private void createDatabase() throws Exception {
   private void createTables() throws Exception {
     for (String tablename : MysqlTpchSchema.schemaMap.keySet()) {
       String sql = "create table " + MysqlTpchSchema.databaseName + "." + tablename + " (";
-      for (String field : MysqlTpchSchema.schemaMap.get(tablename).keySet()) {
-        sql += field + " " + MysqlTpchSchema.schemaMap.get(tablename).get(field) + ", ";
-      }
 
-      sql += "primary key ( ";
-      int index = 1;
-      for (String key : MysqlTpchSchema.primaryKeyMap.get(tablename)) {
-        sql += key;
-        if (index++ < MysqlTpchSchema.primaryKeyMap.get(tablename).size()) {
-          sql += ", ";
-        }
-      }
-      sql += ")";
+      StringJoiner fields = new StringJoiner(", ", "", "");
+      MysqlTpchSchema.schemaMap.entrySet().forEach(field -> fields.add((CharSequence) field));
```

Comment:
Do you need the cast to CharSequence here? Also - since you're using an entrySet() it would return a pair. I don't believe this code works as represented here?

Reply:
Thanks for noticing. fixed this.